homeassistant/components/device_automation/entity.py

```diff
@@ -64,7 +66,7 @@ async def _async_get_automations(
     device_id: str,
     automation_templates: list[dict[str, str]],
     domain: str,
-) -> list[dict[str, str]]:
+) -> GetAutomationsResult:
```

Comment:
I would like to revert this, for reasoning and the review see: <https://github.com/home-assistant/core/pull/72163#issuecomment-1133865649>

Reply:
I have reworked this to remove the incorrect GetAutomation*** aliases